coordinator-tool/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.3
+
+- `new-election` subcommand: Support supplying candidate metadata as paths on the host machine.
```

Comment:
You also removed the manifest and parameters file options.

Maybe we refer to that from the README?